9.4/Dockerfile.rhel7

```diff
@@ -70,6 +72,9 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
 
 VOLUME ["/var/lib/pgsql/data"]
 
+# {APP_DATA} needs to be accessed by postgres user while s2i assembling
+RUN chown -R 26:0 ${APP_DATA} && \
+    chmod -R og+rwx ${APP_DATA}
```

Comment:
Should others have `+rwx` too?

Reply:
I don't think I understand. Do you suggest in rhel image different approach from the one in centos image? https://github.com/sclorg/postgresql-container/pull/208#discussion_r153995545

Reply:
I always mistakenly substitute s/owner/o/ instead of s/user/u/.  Sorry for previous bad advice..

Reply:
So the answer actually :-) no -> rhel/centos should do the same thing from this POV, I'm just asking whether "others" need to have the `rwx` permissions (IMO it shouldn't be needed).  This is security question...